Add Initial CSV to Parquet Conversion, Documentation, and Workflows (#10)
cytomining/pycytominer-transform: docs/source/development.md

# Development

```{mermaid}
flowchart LR
    subgraph Dagger
        containers["Container(s)"]
    end
    subgraph Poetry
        python[Python]
    end
    python --> |running within| containers
```

> Pcytominer-transform is primarily written in Python with related environments managed by Python [Poetry](https://python-poetry.org/).

Reviewer: ```suggestion
Pycytominer-transform is primarily written in Python with related environments managed by Python [Poetry](https://python-poetry.org/).
```

> [Dagger](https://docs.dagger.io/) is used to help create consistent testing locally and with automated tests via containers.

Reviewer: ```suggestion
We use [Dagger](https://docs.dagger.io/) for consistent local testing and for automated tests via containers.
```

Please write in active (instead of passive) voice

Author: Thank you for catching these! I'll update with the suggestions and do some editing as well.

## Getting started

> Local development may be enabled using roughly the following steps.

Reviewer: ```suggestion
To enable local development, perform the following steps.
```

Switching to active voice
